owncloudApp/src/main/java/com/owncloud/android/presentation/files/removefile/RemoveFilesDialogFragment.kt

```diff
@@ -90,44 +75,78 @@ class RemoveFilesDialogFragment : ConfirmationDialogFragment(), ConfirmationDial
                 }
             }
 
-            messageStringId = if (files.size == 1) {
+            handleThumbnail(targetFiles, dialogRemoveThumbnail)
+
+            messageStringId = if (targetFiles.size == 1) {
                 // choose message for a single file
-                val file = files.first()
+                val file = targetFiles.first()
+                messageArguments = file.fileName
```

Comment:
We can delete  `file` variable and use `files.first().fileName` on the value of `messageArguments`

Reply:
In this case I would leave it with my approach. If you notice, the `file` variable is later used to know if the file is a folder, therefore we cannot do without it.
Here the code:
```
val file = targetFiles.first()
                messageArguments = file.fileName
                if (file.isFolder) {
                    R.string.confirmation_remove_folder_alert
                } else {
                    R.string.confirmation_remove_file_alert
                }
```